src/ScreenshotManager.vala

```diff
@@ -92,6 +93,18 @@ namespace Gala
 			throw new DBusError.FAILED ("SelectArea not implemented");
 		}
 
+		public static void create_dir_if_missing (string path)
```

Comment:
This does not need to be static.

Reply:
`save_image` is static, without `create_dir_if_missing` being static to compilation fails with
>ScreenshotManager.vala:117.5-117.32: error: Access to instance member `Gala.ScreenshotManager.create_dir_if_missing' denied
>				create_dir_if_missing (path);